buddypress/groups/single/members-loop.php

```diff
@@ -30,16 +31,17 @@
 
 	<?php bp_nouveau_group_hook( 'before', 'members_list' ); ?>
 
-	<ul id="members-list" class="<?php bp_nouveau_loop_classes(); ?>">
+	<script> 
+		jQuery(document).foundation();
+	</script>
 
-		<?php
+	<?php
+	$count = groups_get_current_group()->total_member_count;
+	If ($count < 22) : ?>
```

Comment:
I guess PHP isn't case-sensitive? Normally I'd say that "if" should be lowercase.

Reply:
Good catch. Apparently not. :-) 

More specifically, it looks like variables and constants are case-sensitive while functions and methods aren't.

Nevertheless, good form to keep all the functions as lower-case. I've fixed this in my local version and will include the fix in the next push. I did a global search in the theme and found only this one instance of `If ( `. I'll try to be conscious of this in the future.

Reply:
Sounds good :-)